.travis.yml

```diff
@@ -31,8 +31,9 @@ before_install:
   - sudo apt-get -q update
 
 install:
-  - sudo apt-get install -y python
+  - sudo apt-get install -y python python3 python3-pip
   - pip install grpcio scapy codecov
+  - pip3 install grpcio coverage
```

Comment:
Oh... then perhaps `scapy-python3` is the package we need to install (https://github.com/phaethon/scapy)...

Also `bessctl/bessctl -- daemon start -- run testing/run_module_tests` in `.travis.yml` now needs "python2" explicitly, as it does use scapy?